Documentation/FrameworkOverview.md

```diff
@@ -8,65 +8,126 @@ learning about new modules and finding more specific documentation.
 For examples and help understanding how to use RAC, see the [README][] or
 the [Design Guidelines][].
 
-## Streams
-
-A **stream**, represented by the [RACStream][] abstract class, is any series of
-object values.
-
-Values may be available immediately or in the future, but must be retrieved
-sequentially. There is no way to retrieve the second value of a stream without
-evaluating or waiting for the first value.
-
-Streams are [monads][]. Among other things, this allows complex operations to be
-built on a few basic primitives (`-bind:` in particular). [RACStream][] also
```

Comment:
And a similar one for `SignalProducer`, mentioning that any `Signal` operator can be `lift`ed to producers.